ci/travis/install-bazel.sh

```diff
@@ -121,3 +121,24 @@ build --remote_upload_local_results=false
 EOF
   fi
 fi
+
+if [ "${TRAVIS-}" = true ]; then
+  # Inject the wrapper function to bashrc, so in future invocation
+  # we get a shorthand to export logs to a temp file on each
+  # test/build event.
+  read -r -d '' bazel_wrapper_func <<-'END'
+bazel() {
+  case "${1-}" in
+    --*) echo "Bazel startup options not yet implemented for this thunk" 1>&2; false;;
+    test) command bazel "$1" --build_event_json_file "$(mktemp /tmp/bazel_event_logs/bazel_log.XXXXX)" "${@:2}";;
+    *) command bazel "$@";;
+  esac
+}
+END
```

Comment:
```suggestion
  bazel() {
    case "${1-}" in
      --*) echo "Bazel startup options not yet implemented for this thunk" 1>&2; false;;
      test) command bazel "$1" --build_event_json_file "$(mktemp /tmp/bazel_event_logs/bazel_log.XXXXX)" "${@:2}";;
      *) command bazel "$@";;
    esac
  }
  declare -f bazel >> ~/.bashrc
```
You can take out the `read` and just use `declare -f`.